apps/src/studio/Item.js

```diff
@@ -45,7 +45,14 @@ Item.prototype.getDirectionFrame = function() {
   // assign a new displayDir from state table; only one turn at a time.
 
   if (this.dir !== this.displayDir && this.displayDir !== undefined) {
-    if (Studio.tickCount && (0 === Studio.tickCount % 2)) {
+    // Special case: if the item has now stopped, just switch to idle
```

Comment:
why do you need this special case? can't they just turn and then go idle? (I thought playlab already did that)